pkg/machine/applehv/machine.go

```diff
@@ -101,15 +100,7 @@ func checkProcessRunning(processName string, pid int) error {
 // is assumed GiB
 func resizeDisk(mc *vmconfigs.MachineConfig, newSize strongunits.GiB) error {
 	logrus.Debugf("resizing %s to %d bytes", mc.ImagePath.GetPath(), newSize.ToBytes())
-	// seems like os.truncate() is not very performant with really large files
-	// so exec'ing out to the command truncate
-	size := fmt.Sprintf("%dG", newSize)
-	c := exec.Command("truncate", "-s", size, mc.ImagePath.GetPath())
-	if logrus.IsLevelEnabled(logrus.DebugLevel) {
-		c.Stderr = os.Stderr
-		c.Stdout = os.Stdout
-	}
-	return c.Run()
+	return os.Truncate(mc.ImagePath.GetPath(), int64(newSize.ToBytes()))
```

Comment:
I'm all for it, but I'd split that to its own commit. At some point, we can add a vfkit helper to do this even if it's a one liner.